src/main/java/nl/hsac/fitnesse/fixture/util/selenium/driverfactory/LocalDriverFactory.java

```diff
@@ -49,6 +50,9 @@ public WebDriver createDriver() {
             } else if ("internetexplorerdriver".equalsIgnoreCase(driverClass.getSimpleName())) {
                 InternetExplorerOptions ieOptions = getInternetExplorerOptions(profile);
                 driver = new InternetExplorerDriver(ieOptions);
+            } else if ("edgedriver".equalsIgnoreCase(driverClass.getSimpleName())) {
```

Comment:
This will add another deprecation warning to the build. Do you see how hard/easy it would be to switch to EdgeOptions (and ChromeOptions). I would appreciate another PR for that...

Reply:
I had a quick look, but I didn't see an easy solution. I hereby promise to look into it and do a new PR somewhere later this month :-)

However we have a current need for chromium capabilities in the chromium version of edge, so if this could act as an intermediate solutions that would be very helpful